docs/platform-services/automation-service/app-central/integrations/smtp-v3.md

```diff
@@ -19,6 +19,21 @@ Allows you to connect your mailbox with Cloud SOAR and send emails via SMTP prot
 
 * **Send Email** (*Notification*) - Send an email.
 
+## Guidelines for Composing the Email Body in the "Send Email" Action
+
+1. Raw Text Message:
+When sending a plain text message in the email body, it's important to avoid clicking on the text field after you've written your message. If you do click on the text field, the formatting of your text may become distorted or "munged" together. To ensure your text retains its original formatting when delivered in the email, avoid interacting with the text field after composing your message.
+
+2. HTML Template:
+For emails using an HTML template, ensure you have a properly structured HTML template. Whether you click on the text field or not after writing the message, the email will generally display correctly with proper formatting as expected.
+
+However, for optimal results with HTML templates, we recommend clicking on the text field after writing your message. This action helps ensure that the HTML content is recognized and formatted correctly, giving you the desired outcome in the final email.
+
+## Summary
+
+* **Raw Text Message** - Avoid clicking on the text field after writing to prevent formatting issues.
+* **HTML Template** - Clicking on the text field after writing is recommended for the best formatting, though your email should still display correctly without this step.
```

Comment:
```suggestion
* **Raw Text Message**. Avoid clicking on the text field after writing to prevent formatting issues.
* **HTML Template**. Clicking on the text field after writing is recommended for the best formatting, though your email should still display correctly without this step.
```